### PR TITLE
Added in search functionality for courses and course-runs on the djan…

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
…go admin page for course catalog. ECOM-4705.

@edx/ecommerce please review.

### CODE REVIEW

File: course_discovery/settings/local.py [hunk 1]
```diff
@@ -48,9 +48,9 @@
 
 # AUTHENTICATION
 # Set these to the correct values for your OAuth2/OpenID Connect provider (e.g., devstack)
-SOCIAL_AUTH_EDX_OIDC_KEY = 'replace-me'
-SOCIAL_AUTH_EDX_OIDC_SECRET = 'replace-me'
-SOCIAL_AUTH_EDX_OIDC_URL_ROOT = 'replace-me'
+SOCIAL_AUTH_EDX_OIDC_KEY = '4f71ee91afa30bfb14d2'
```
Comment (schenedx) [inline course_discovery/settings/local.py]: You should put these changes to a `private.py` file in your local machine. That file gets ignored. Please revert your changes here.

**Comment by clintonb** [comment]
👍 once changes to local.py removed.

File: course_discovery/settings/local.py [hunk 2]
```diff
@@ -50,7 +50,7 @@
 # Set these to the correct values for your OAuth2/OpenID Connect provider (e.g., devstack)
 SOCIAL_AUTH_EDX_OIDC_KEY = 'replace-me'
 SOCIAL_AUTH_EDX_OIDC_SECRET = 'replace-me'
-SOCIAL_AUTH_EDX_OIDC_URL_ROOT = 'replace-me'
+SOCIAL_AUTH_EDX_OIDC_URL_ROOT = 'http://127.0.0.1:8000/oauth'
```
Comment (clintonb) [inline course_discovery/settings/local.py]: This still needs to be reverted.

Comment (clintonb) [inline course_discovery/settings/local.py]: Local users should define all of these in `private.py`. Not everyone hosts LMS at the same location.

**Comment by clintonb** [comment]
Use git rebase to squash these into a single commit.

**Comment by rlucioni** [comment]
🎉

**Pull request merged**